Examples/mlfit-Ex.Rout.save

```diff
@@ -17,22 +17,22 @@ Type 'demo()' for some demos, 'help()' for on-line help, or
 'help.start()' for an HTML browser interface to help.
 Type 'q()' to quit R.
 
-> pkgname <- "MultiLevelIPF"
```

Comment:
Do we still need this file?

Reply:
I agree snapshot tests with testthat 3 are much better. Would you like to take a look?

Reply:
Sure

Reply:
I checked our unit tests, I don't think we need snapshot tests anywhere. Do we?

Reply:
Maybe for the printed output?

Reply:
Changing to `testthat edition 3` is going to breaks a few things.
- `expect_equal()` now has tolerance of `.Machine$double.eps^0.5`, which can break our expect_equal tests by chance.
- `expect_is()` is deprecated.

Will investigate further where it would be good for us to use `expect_snapshot()`.